Adding All Day DevOps, Global SKILup Day, & DevOps Summit

MikeRosTX opened the pull request:
Requesting this PR be merged for the 24hrs-FREE-Online **_All Day DevOps_**.

finspin commented:
Mike, thanks for your contribution! I removed the SKILup events for now but would love to add them back in the future. The reasons for removing are the following:

* all SKILup events have already taken place, the only upcoming event in London doesn't contain any details as of now
* I would prefer to add these events separately, one per each city so that gotodevops.org visitors can utilize search filters

The other 2 events have been added.

MikeRosTX commented:
But the SKILup event I added is a Global SKILup Day that is happening online similar to All Day DevOps.  We can talk more about it on our call next week with Arlene Hoffman of the DevOps Institute.
Cheers,
Mike Rosado
LinkedIn | Twitter | GitHub | Facebook
Co-Organizer - DevOps Live Meetup
Global Core Organizer - devopsdays.org 

    On Friday, September 6, 2019, 01:53:19 PM CDT, Jaro Spisak <notifications@github.com> wrote:  
 
 
Mike, thanks for your contribution! I removed the SKILup events for now but would love to add them back in the future. The reasons for removing are the following:
   
   - all SKILup events have already taken place, the only upcoming event in London doesn't contain any details as of now
   - I would prefer to add these events separately, one per each city so that gotodevops.org visitors can utilize search filters

The other 2 events have been added.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.
